pymodm/queryset.py

```diff
@@ -220,6 +220,33 @@ def order_by(self, ordering):
         clone._order_by = ordering
         return clone
 
+    def project(self, projection):
+        """Specify a raw MongoDB projection to use in QuerySet results.
+
+        This method is *not* chainable and overrides any projections created
```

Comment:
I mean "not chainable with other projection methods", since it will wipe them all out. Updated docs language.